src/test/fuzz/block_index.cpp

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2023 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <chain.h>
+#include <chainparams.h>
+#include <node/blockstorage.h>
+#include <txdb.h>
+#include <validation.h>
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+#include <test/util/setup_common.h>
+
+namespace {
+
+const BasicTestingSetup* g_setup;
+
+// Hardcoded block hash and nBits to make sure the blocks we store pass the pow check.
+const uint256 g_block_hash{uint256S("000000002c05cc2e78923c34df87fd108b22221ac6076c18f3ade378a4d915e9")};
+uint32_t g_bits{0x1d00ffff};
```

Comment:
nit: I think constants are not supposed to be prefixed with `g_`

```suggestion
const uint256 g_block_hash{uint256S("000000002c05cc2e78923c34df87fd108b22221ac6076c18f3ade378a4d915e9")};
const uint32_t g_bits{0x1d00ffff};
```

I think you could also just use the genesis block (`Params().GenesisBlock()`) for these two values.

Reply:
> nit: I think constants are not supposed to be prefixed with g_

https://github.com/bitcoin/bitcoin/blob/3ba8de1b704d590fa4e1975620bd21d830d11666/doc/developer-notes.md?plain=1#L87-L93

> I think you could also just use the genesis block (Params().GenesisBlock()) for these two values.

Done.